assets/src/edit-story/components/library/panes/media/common/innerElement.js

```diff
@@ -136,20 +136,28 @@ function InnerElement({
     width: width,
     height: height,
     alt: alt,
-    'aria-label': alt,
+    crossOrigin: 'anonymous',
   };
-  const cloneProps = {
+
+  const commonImageProps = {
     ...commonProps,
+    onLoad: makeMediaVisible,
     loading: 'lazy',
+  };
+
+  const cloneProps = {
+    ...commonImageProps,
```

Comment:
> This just makes the image props between poster image and image the same.

@spacedmonkey `cloneProps` shouldn't really need `onLoad`, what was the reason for adding it there as well? Clone is the visible clone image while dragging media from the library onto a page, it only becomes visible once starting dragging an image/video. Thoughts?

Reply:
Definitely looks like a bug. Reverted in d7b236efb7ba2f7515509d9e2afe11ad791b6882